doc/source/building/windows.rst

```diff
@@ -8,155 +8,230 @@ Building From Source on Windows
 Overview
 --------
 
-Compared to OSX and Linux, building NumPy and SciPy on Windows is more difficult,
-largely due to the lack of compatible, open-source libraries like LAPACK_ or
-ATLAS_ that are necessary to build both libraries and have them perform
-relatively well. It is not possible to just call a one-liner on the command
-prompt as you would on other platforms via ``sudo apt-get install`` machinery.
-
-This document describes one option to build OpenBLAS and SciPy from source that 
-was validated for scipy 1.0.0. However, in light of all the work currently being 
-done, **do not expect** these instructions to be accurate in the long-run and be
-sure to check up on any of the open source projects mentioned for the most up-to-date
-information. For more information on all of these projects, the Mingwpy_ website
-is an excellent source of in-depth information than this document will provide.
+Compared to OSX and Linux, building NumPy and SciPy on Windows is more 
+difficult, largely due to the lack of compatible, open-source libraries like 
+LAPACK_ or ATLAS_ that are necessary to build both libraries and have them 
+perform relatively well. It is not possible to just call a one-liner on the 
+command prompt as you would on other platforms via ``sudo apt install`` 
+machinery. 
+
+This document describes one option to build OpenBLAS and SciPy from source 
+that was validated for scipy 1.4.0. However, in light of all the work 
+currently being done, please **do not expect** these instructions to be accurate
+in the long-run and be sure to check up on any of the open source projects
+mentioned for the most up-to-date information. For more information on all of
+these projects, the Mingwpy_ website is an excellent source of in-depth 
+information than this document will provide.
 
 .. _Mingwpy: https://mingwpy.github.io/
-.. _ATLAS: http://math-atlas.sourceforge.net/
```

Comment:
Looks like CircleCI failure is because of this deletion. `ATLAS_` is still used.

Reply:
Ah good catch! Thanks.